source/agora/registry/API.d

```diff
@@ -43,7 +43,7 @@ public struct RegistryPayload
     public Signature signature;
 
     ///
-    public void signPayload (in SecretKey secret_key) nothrow
+    public void signPayload (in KeyPair secret_key) nothrow
```

Comment:
`key_pair` would be better for the parameter.

Reply:
Good catch! Will go with `kp` though, as it's a common abbreviation.